index.js

```diff
@@ -97,11 +99,34 @@ module.exports = {
     'import/order': [
       'error',
       {
-        groups: ['builtin', 'external', 'parent', 'sibling', 'index'],
+        groups: [
+          'builtin',
+          'external',
+          'parent',
+          'sibling',
+          'index',
+          'object',
+          'type',
```

Comment:
`2.24.0` added the ability to specify `type`. We use a custom plugin to achieve the same internally, it would be great to deprecate this and use `import/order`'s native support.

https://github.com/import-js/eslint-plugin-import/blob/v2.24.0/CHANGELOG.md#2230---2021-05-13